tests/python/relay/test_pass_annotate_target.py

```diff
@@ -738,8 +738,8 @@ def after():
         mod = tvm.IRModule.from_expr(func)
         return mod
 
-    for annotate_non_call_ops in [True, False, True]:
-        result = transform.AnnotateTarget(target)(before())
+    for annotate_non_call_ops in [True, False]:
+        result = transform.AnnotateTarget(target, annotate_non_call_ops)(before())
```

Comment:
Changed this test because it looked incorrectly, please correct me if there was a reason the variable wasn't used

Reply:
Good catch. Looks like a miss.